MAiD - Case Management App/force-app/main/default/layouts/Form_1634__c-Form 1634 Layout.layout-meta.xml

```diff
@@ -645,10 +649,6 @@
                 <behavior>Edit</behavior>
                 <field>An_agreement_with_the_patient_to_waive_f__c</field>
             </layoutItems>
-            <layoutItems>
-                <behavior>Edit</behavior>
-                <field>If_Yes_patient_consented__c</field>
```

Comment:
@ahshaik1: This field was not deleted by Rana or Amish. But it looks like you removed it, can you please why this is the case? I pulled all the latest dev changes into this branch. Can you fix this by putting the field back and pushing another commit into this same branch?

Reply:
@cgiSunilM I have done the needed changes.. this issue was because i deployed before doing pull.. i will make this dosent happen again